mix.exs

```diff
@@ -44,6 +44,7 @@ defmodule GRPC.Mixfile do
       # but we can't depend on an RC for releases
       {:gun, "~> 2.0.1", hex: :grpc_gun},
       {:cowlib, "~> 2.11"},
+      {:ranch, "~> 2.0", override: true},
```

Comment:
Why do we need the override?
We can't publish the package with the override here :(

Reply:
because cowlib still reference to `"1.8.0"`, this means this approach need to suppress warning from dialyzer on no_match for `:ranch.child_spec`

Reply:
I'll revert it back

Reply:
Ok, so we can't update ranch after all.
Best approach is to revert to the case implementation and ignore the dialyzer warning for that specific line.

Reply:
Hex blocks packages that have override: true for any dependencies, because override should only be used at the top level